cmd/trust-manager/app/app.go

```diff
@@ -74,12 +75,16 @@ func NewCommand() *cobra.Command {
 			eventBroadcaster.StartLogging(func(format string, args ...any) { mlog.V(3).Info(fmt.Sprintf(format, args...)) })
 			eventBroadcaster.StartRecordingToSink(&clientv1.EventSinkImpl{Interface: cl.CoreV1().Events("")})
 
+			renewDeadline := time.Second * 10
+			leaseDuration := time.Second * 15
```

Comment:
While the tests are now green, we are not using the new flags. 😁 Can you try to fix this please, @jabdoa2?

Reply:
Ups. Left over from my tests. Will fix that. Sorry